src/backend/base/langflow/exceptions/api.py

```diff
@@ -1,2 +1,20 @@
+from fastapi import HTTPException
+from pydantic import BaseModel
+
+
 class InvalidChatInputException(Exception):
     pass
+
+
+# create a pidantic documentation for this class
+class exceptionBody(BaseModel):
+    message: str | list[str]
+    traceback: str | list[str] | None = None
+    description: str | list[str] | None = None
+    code: str | None = None
+    suggestion: str | list[str] | None = None
+
```

Comment:
Maybe this instead of the `get_api_exception_body`
```suggestion
	def from_exc_and_flow(exc, flow): ...
```

Reply:
what about `@staticmethod`?